### DISCUSSION TITLE
Change a table name in all measures

### DISCUSSION THREAD

**Discussion by jjr333** [DaxStudio/DaxStudio#651] [Q&A] [answered]
I have added a new data table to my model.
Now I would like to use Search & Replace in all measures to have them point to the new table.

What steps to I need to take?

Cheers,
Julian

**Comment by dgosbell** [comment, marked as answer]
The simplest thing would probably be to swap the names of your two tables. DAX Studio is query / performance tuning tool which only makes a read-only connection to the data model - to make edits to the model you would need to use a tool like [Tabular Editor](https://tabulareditor.com/). Tabular Editor 2 is free and open source. Tabular Editor 3 is a paid product, but they do have a free 30 day trial. 

In Tabular Editor 3 you can right click on your model and open a DAX Script window where you can see all your measures in the one long script so it should be easy to do a global search and replace from there.

**Reply by jjr333** [reply]
Thank you.
